test/Parse/multiline_errors.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: %target-typecheck-verify-swift
+
+import Swift
+
+// ===---------- Multiline --------===
+
+_ = """
+    Eleven
+  Mu
+    """ // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
+_ = """
+	Twelve
+\tNu
+	""" // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
+_ = """
+  Thirteen
+	Xi
+  """ // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
```

Comment:
I assume this is because of a tab character? Please add a comment so that it's also clear to whoever reads these tests why it's failing.

Any such comments are good spots for a more specific or targeted diagnostic (again, shouldn't block this PR).